elements/bulbs-video/components/revealed.js

```diff
@@ -202,7 +202,7 @@ export default class Revealed extends React.Component {
   }
 
   makeVideoPlayer (element, videoMeta) {
-    element.id = jwPlayerIdCounter++;
+    element.id = `jw-player-${jwPlayerIdCounter++}`;
```

Comment:
It does require it, but IDs are supposed to be unique on the page and I don't know the internals of how the jwplayer library handles it.  I think their library maintainers are likely safely assuming the ID handed to them is unique on the page.